### PR TITLE
Updating seq-reject-2.2.1 and seq-reject-2.2.0 to add clarity regardi…

### PR DESCRIPTION
**Pull request by elnyry**
…ng required fields and message formats for failure paths

### CODE REVIEW

File: Diagrams/SequenceDiagrams/seq-reject-2.2.1.plantuml [hunk 1]
```diff
@@ -275,6 +301,32 @@ group Fulfil Handler Consume (Failure)
...
+                            metadata: {
+                                event: {
+                                    id: <uuid>,
+                                    responseTo: <previous.uuid>,
+                                    type: position,
+                                    action: abort,
+                                    createdAt: <timestamp>,
+                                    state: {
+                                        status: "fail",
+                                        code: 0
```
Comment (ggrg) [inline Diagrams/SequenceDiagrams/seq-reject-2.2.1.plantuml]: Shouldn't code be a positive number?

File: Diagrams/SequenceDiagrams/seq-reject-2.2.1.plantuml [hunk 2]
```diff
@@ -220,8 +220,34 @@ group Fulfil Handler Consume (Failure)
...
+                            metadata: {
+                                event: {
+                                    id: <uuid>,
+                                    responseTo: <previous.uuid>,
+                                    type: position,
+                                    action: reject,
+                                    createdAt: <timestamp>,
+                                    state: {
+                                        status: "fail",
+                                        code: 0
```
Comment (ggrg) [inline Diagrams/SequenceDiagrams/seq-reject-2.2.1.plantuml]: Shouldn't code be a positive number?

File: Diagrams/SequenceDiagrams/seq-reject-2.2.1.plantuml (hunk 1 above)
Comment (elnyry) [reply Diagrams/SequenceDiagrams/seq-reject-2.2.1.plantuml]: Updated

File: Diagrams/SequenceDiagrams/seq-reject-2.2.1.plantuml (hunk 2 above)
Comment (elnyry) [reply Diagrams/SequenceDiagrams/seq-reject-2.2.1.plantuml]: Updated

**Pull request merged**